boards/xtensa/esp32/doc/esp32.rst

```diff
@@ -62,6 +62,46 @@ The toolchain is available for Linux, Windows, and Mac hosts and
 instructions to obtain and set them up are available in the ESP-IDF
 repository, using the toolchain and SDK links above.
 
+Docker container based builds
+=============================
+
+You can skip locally installing and configuring the Zephyr build system by
+using pre-made the ``svendowideit/zephyr-esp32`` Docker image.
+
+For example, to build the ``hello_world`` sample and flash it when its attached to ttyUSB0:
+
+.. code-block:: console
+
+   $ docker run --rm -it --device /dev/ttyUSB0:/dev/ttyUSB0 -w /esp32 svendowideit/zephyr-esp32 sh -c "cmake -DBOARD=esp32 /zephyr/samples/hello_world && make flash"
+
+You can test that the ESP32 has hello_world running by attaching a serial console
+to ttyUSB0, and resetting the device:
+
+.. code-block:: console
+   $ screen /dev/ttyUSB0 115200
+   ts Jun  8 2016 00:22:57
```

Comment:
is something truncated here (does the output line start with "ts" here and "ets" a few lines down?

Reply:
I tested this a few times, and this output is pretty consistent - if its a bug, then it can be updated later :D